packages/react/runtime/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "@preact/signals-react-runtime",
+  "description": "Sub package for @preact/signals-react that contains the pure runtime functions",
+  "private": true,
+  "amdName": "reactSignalsRuntime",
+  "main": "dist/runtime.js",
+  "module": "dist/runtime.module.js",
+  "unpkg": "dist/runtime.min.js",
+  "types": "dist/index.d.ts",
+  "source": "src/index.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "browser": "./dist/runtime.module.js",
+      "umd": "./dist/runtime.umd.js",
```

```suggestion
      "umd": "./dist/runtime.min.js",
```

I recognize you copied from our other packages, but they look to have been misconfigured too.

Alternatively, we could just remove it. We have yet to see a complaint from this broken configuration in the other packages and I don't know of any tools that will actually consume `exports.umd`.

---

Oh good catch. I'll fix them.

---

Alternatively, #373. Not sure which is better

---

Oh whoops! Just saw that. We can do that.